code/scripts/templates/monitconfig.cfg

```diff
@@ -0,0 +1 @@
+${TORQHOME}/appconfig/process.csv     ${TORQHOME}/torq.sh     monitconfig.cfg
```

Comment:
This + `fill_templates.sh` was set up by me a long time ago rather naively, there's probably better ways to do it and definitely better ways to set up this config file, would be worth looking into it.